### DISCUSSION TITLE
Is possible to remove the misidentified twin boundaries in MTEX?

### DISCUSSION THREAD

**Discussion by Drbiaobiao** [mtex-toolbox/mtex#1448] [Ask Anything]
Hello,
This is Biaobiao.
When I tried to use the code to analyze the twinning boundaries of Mg, I found some grain boundaries, actually not twinning boundaries, are presented as twinning boundaries (see attached example file).
Well, I understand the misorientation angle and misorientation axis of these grain boundaries are just near them of twinning boundaries. They are kind of misidentified by code.
Here is my question, could we eliminate the misidentified ones and just give the true twinning boundaries?
Thanks for your time.
Best wishes,
Biaobiao
![Example](https://user-images.githubusercontent.com/58214481/171652729-c72cffb4-3912-4cb6-a699-2e6269cdb0cc.png)

**Comment by kilir** [comment]
Hi,
how would you discriminate the wrongfully selected boundaries? Also, the relevant lines of code might help.
Cheers,
Rüdiger
